homeassistant/components/switch/rainbird.py

```diff
@@ -0,0 +1,116 @@
+"""
+Support for Rain Bird Irrigation system LNK WiFi Module.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/rainbird/
+"""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.const import (CONF_PLATFORM, CONF_SWITCHES, CONF_ZONE,
+                                 CONF_FRIENDLY_NAME, CONF_TRIGGER_TIME,
+                                 CONF_SCAN_INTERVAL, CONF_HOST, CONF_PASSWORD)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.exceptions import PlatformNotReady
+
+REQUIREMENTS = ['pyrainbird==0.1.0']
+
+DOMAIN = 'rainbird'
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
Extend the schema -> https://home-assistant.io/developers/code_review_platform/#3-configuration